chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -403,6 +403,7 @@ def make_zone(zone)
   command "nscd -i hosts"
   action :nothing
   subscribes :run, "template[/etc/bind/db.#{node[:dns][:domain]}]"
+  only_if { File.exist?("/var/run/nscd/nscd.pid") }
```

Comment:
I checked that the .pid file exists on SLE11 and SLE12 iff nscd is running. having nscd installed is not enough.